content/en/docs/developer/e2e-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+draft: false
+linktitle: E2E Test Guide
+menu:
+  docs:
+    parent: developer guide
+    weight: 15
+title: Run E2E test
+toc: true
+type: docs
+
+---
+End-to-End (E2E) testing is a crucial component in modern software development, designed to simulate user interactions across the entire application to ensure all components and integrations function together seamlessly. By incorporating E2E testing, we can verify that changes in code do not disrupt existing functionalities, maintaining the integrity and reliability of the system even as it evolves. 
+
+## Prerequisites
+
+The Go version in go.mod is 1.22.0, it is necessary to install go 1.22.
```

Comment:
Just add `- Go` before `- Docker` below. They would update their version if it's not appropriate.